Feliz.CompilerPlugins/ReactComponent.fs

```diff
@@ -0,0 +1,93 @@
+﻿namespace Feliz
+
+open Fable
+open Fable.AST
+
+/// <summary>Transforms a function into a React function component. Make sure the function is defined at the module level</summary>
+type ReactComponentAttribute() =
+    inherit MemberDeclarationPluginAttribute()
+    override _.FableMinimumVersion = "3.0"
+
+    /// <summary>Transforms call-site into createElement calls</summary>
+    override _.TransformCall(compiler, memb, expr) =
+        let membArgs = memb.CurriedParameterGroups |> List.concat
+        match expr with
+        | Fable.Call(callee, info, typeInfo, range) when List.length membArgs = List.length info.Args ->
+            let callee =
+                match callee with
+                | Fable.Expr.IdentExpr ident ->
+                    // capitalize same-file references
+                    Fable.Expr.IdentExpr { ident with Name = AstUtils.capitalize ident.Name }
+                | Fable.Expr.Import(importInfo, fableType, sourceLocation) ->
+                    // capitalize component imports from different modules/files
+                    let selector =
+                        match importInfo.Selector with
```

Comment:
Ah, forgot this is `Expr`, it should actually be a string. I think I did it like that because in some situations we were adding string literals which are resolved later. But it complicates things, so I've just changed `ImportInfo.Selector` (and Path) to be string. Can you please upgrade to latest Fable.AST? This was the last change before the beta release so we need to be more careful now! :)